rules/windows/c2_rundll32_sequence.toml

```diff
@@ -22,16 +22,20 @@ tags = ["Elastic", "Windows"]
 type = "eql"
 
 query = '''
+/* replace winlog.event_data.OriginalFileName with process.pe.original_file_name once field is available in winlogbeat */
```

Comment:
Just an idea. We could use #TODO's in our rules and include the corresponding issue number in the comment to track these type of follow up actions

Reply:
rdub and I have talked about how to preserve comments like this, but you are right, just leaving them as `TODO` comments in the TOML may be the perfect spot, since we don't need the comments to flow to Kibana 👍

Reply:
only downside, `toml-lint` rips out toml comments

Reply:
but yeah, i agree on the comments not going to kibana.
i could take a stab at adding a function to endgameinc/eql to strip comments. would that work?

Reply:
could we strip specific comments such as with a specific leading comment like 

```python
/* REPO: this comment will
be
removed 
in its entirety 

*/

/* but this one will
not
*/
```